templates/aws-stack.yml

```diff
@@ -626,10 +632,13 @@ Rules:
             - !Equals
```

Comment:
I think this assertion is wrong – it'll pass if you provide PipelineSigningKMSKeyId and PipelineSigningKMSKeyArn, without PipelineSigningKMSKeySpec.

Reply:
From my understanding, this only verifies that one of these are set. It wasn't every validating the "but not both" portion. Will re-work this a little